cookbook/how_to_apply_patch.rst

```diff
@@ -0,0 +1,122 @@
+.. index::
+    single: Patch
+
+How to apply patch
+==================
+
+The patch file name supports the following naming convention: {package}-{version_to_apply}.patch.
+For example, platform-1.9.2.patch.
+
+**1** Create backups of your Database and Code.
+
+**2** Stop the cron tasks
+
+.. code-block:: bash
+
+    $ crontab -e -uwww-data
+
+    .. note::
+        www-data can be changed to user under which your web server runs
+
+Comment this line.
+
+.. code-block:: text
+
+     */1 * * * * /usr/bin/php /srv/prod/crm-application/app/console --env=prod oro:cron >> /dev/null
```

Comment:
Let's use "/path/to/application" everywhere instead of "/srv/prod/crm-application".